Use underlying system random instead of homebrew random

pacohope opened the pull request:
This addresses #76.

I'll be honest, I don't understand why this library exists, or why it was doing what it was doing. It was not producing unpredictable randomness. It was the same series of random numbers every time. 
This patch to the library basically passes the calls onward to the `urandom` or `random` module. The output is now as random (or not) as the underlying `urandom` module and its entropy sources.

Pezmc commented:
I'd like to see this merged, particularly because the `upip` modules fall after `lib` in sys.path (`['', '/flash', '/flash/lib', '/flash/upip']`), so we can't use the upip random module

pacohope commented:
@Pezmc I didn't delete the `lib/random.py` file, even though I think that's the right thing to do. Lots of apps are using it. For now, I am just solving the fact that randomness was predictable by having this unnecessary random library use the underlying randomness. I'll see if this library can be completely removed. I'll submit that on a separate branch/pr.

pacohope commented:
Actually, I did it. It wasn't hard. I updated this branch and now the `lib/random.py` file should be removed and all other dependencies on it that I could find are also removed. Tests pass.
